test/living-html/postmessage.js

```diff
@@ -0,0 +1,123 @@
+"use strict";
+
```

Comment:
The script injection is a remnant from when I was originally using separate HTML files for each test, similar to [test/window/frame.js](https://github.com/tmpvar/jsdom/blob/c59ad9a8f20508fc2b199195891ef62999ba7b8c/test/window/frame.js)

Now that the strings are completely static, as you pointed out, this approach doesn't make much sense anymore. Would it be alright if I removed the script injection and reached directly into the iframes for all tests?